sms.module

```diff
@@ -21,7 +24,57 @@ define('SMS_DIR_IN',    2);
 define('SMS_DIR_ALL',   4);
 
 /**
+ * Implements hook_cron().
+ */
+function sms_cron() {
+  /** @var \Drupal\sms\Provider\PhoneNumberProviderInterface $phone_number_provider */
```

Comment:
Can we document what this is doing? Do we need it elsewhere? Does it make sense to factor it out to a helper method or something?